### DISCUSSION TITLE
4-bit PCX image (16 colors)

### DISCUSSION THREAD

**Discussion by Tjossul** [dlemstra/Magick.NET#1162] [Help]
Hello,

sadly I could not create a 4-bit PCX image until now. 

            using (var imagePcx = new MagickImage(imagePath))
            {
                imagePcx.Format = MagickFormat.Pcx;
                imagePcx.ColorType = ColorType.Palette;
                imagePcx.Write(imagePcxPath);
            }

What are the right settings for that?

![1](https://user-images.githubusercontent.com/39447922/158436090-5c1be8ef-a056-4c45-85bb-ef3810d68a59.png)
![2](https://user-images.githubusercontent.com/39447922/158436098-5b67c4cf-6672-46f5-875c-5ee4d4c6026b.png)

As you can see, I am writing the file as a "4-bit" PCX but IrfanView says it is an 8-bit PCX image.

![3](https://user-images.githubusercontent.com/39447922/158436100-cedb5dbd-da7c-4d99-a496-d16f9a6a357d.png)

Thanks in advance!
T.

**Comment by dlemstra** [comment]
ImageMagick currently has no support for 4-bit pcx images.

**Reply by Tjossul** [reply]
Hello,

thanks for your reply.

Do you have plans to implement that?

Regards.
T.

**Comment by dlemstra** [comment]
No plans for that now. But you can request it here: https://github.com/ImageMagick/ImageMagick/issues/new/choose.

**Comment by Tjossul** [comment]
OK, thanks.

Here it is:

https://github.com/ImageMagick/ImageMagick/issues/4981

Regards.
T.
